src/ClientCommand.cpp

```diff
@@ -438,6 +438,50 @@ void CClient::UserCommand(CString& sLine) {
                       "] channels matching [" + sPatterns + "]");
             PutStatus("Disabled [" + CString(uDisabled) + "] channels");
         }
+    } else if (sCommand.Equals("MOVECHAN")) {
+        if (!m_pNetwork) {
+            PutStatus("You must be connected with a network to use this command");
```

Comment:
Missing `t_s()`, and below

I wonder how I missed it some other strings in this file